XLS-0033d-compact-fungible-tokens/README.md

```diff
@@ -824,85 +833,143 @@ A JSON object representing a dictionary of accounts to CFToken objects. Includes
+That said, there is some overlap in functionality between the two. For example, both CFTs and Trustlines can be used to issue a stablecoin. However, the original intent behind CFTs and Trustlines is subtly different, which impacts the on-ledger design of each. For clarity, Trustlines were invented primarily to service the idea of "community credit" and also to enhance liquidity on the ledger by making the same types of currency fungible amongst differing issuers (see [rippling](https://xrpl.org/rippling.html#rippling) for an example of each). CFTs, on the other hand, have three primary design motivations that are subtly different from Trustlines: (1) to enable tokenization using as little space (in bytes) on ledger as possible; (2) to eliminate floating point numbers and floating point math from the tokenization primitive; and (3) to make payment implementation simpler by, for example, removing [rippling](https://xrpl.org/rippling.html#rippling) and allowing CFT usage in places like [escrows](https://xrpl.org/escrow.html#escrow) or [payment channels](https://xrpl.org/payment-channels.html) in more natural ways.
+
+### 2.1.2. Are CFTs meant to replace Trustlines?
+
+No, replacing Trustlines is not the intent behind CFTs. Instead, it's likely that CFTs and Trustline can and will coexist because they enable subtly different use-cases (see [FAQ 4.1](#41-are-cfts-different-from-trustlines), in particular the part about "rippling.").
+
+### 2.1.3. Are CFTs targeted for Mainnet or a Sidechain?
+
+This is still being considered and debated, but is ultimately up to Validators to decide. On the one hand, enabling CFTs on Mainnet would enable some new tokenization use-cases that could be problematic if Trustlines were to be used (see [FAQ 2.1.6](#216-an-early-draft-of-this-cft-proposal-stored-cftokens-in-a-paging-structure-similar-to-that-used-by-nfts-why-was-that-design-abandoned) for more details). On the other hand, adding CFTs introduces a new payment type into the payment engine, which complicates both the implementation of rippled itself, as well as XRPL tooling. 
+
+Regardless, it might be desirable to let CFTs "bake" on a sidechain to prove their utility, and in-tandem try to [optimize trustlines storage](https://github.com/XRPLF/rippled/issues/3866) and even (eliminate Custom Math)[https://github.com/XRPLF/rippled/issues/4120) from trustlines.
```

Comment:
I think it would be good to add that we will first preview the feature on a CFT-Devnet and then perhaps "bake" in a sidechain

Reply:
Good suggestion. Fixed via a2a26e22d668765bd7e635f468a8d19c215cfe46